README.md

```diff
@@ -0,0 +1,172 @@
+<a name="My Portfolio"></a>
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+</div>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [My_Portfolio] <a name="My Portfolio">My Portfolio</a>
+
+> Describe your project in 1 or 2 sentences.
+This is a My Portfolio  project.
+**[your_project__name]** is My Portfolio
+
+## 🛠 Built With <a name="built-with">Html, css</a>
+
+### Tech Stack <a name="tech-stack">HTML5, CSS3</a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+HTML5 and css3 which are the latest version were used in this project.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] Kindly remove these lines after you are done updating the README file. Every line with the `>` prefix is an instruction on what to put there. ✍️